### ISSUE TITLE
Accessibility table for Button should have aria-label applied to .pf-c-button.pf-m-plain

### ISSUE THREAD

**Issue by bitwiseor** [patternfly/patternfly#4193]
In [the accessibility table for the Button component](https://www.patternfly.org/v4/components/button/html#accessibility), the aria-label attribute says it should be applied to `.pf-m-plain`. This needs to be updated, as the aria-label for a Button should be applied to `.pf-c-button.pf-m-plain`.

<img width="852" alt="Screen Shot 2021-06-28 at 9 19 47 AM" src="https://user-images.githubusercontent.com/43662677/123678785-c3272480-d814-11eb-85e8-a372cdb42689.png">
